PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -380,6 +404,20 @@
             image.imageAsset?.value(forKey: "_containingBundle") != nil
         }
 
+        // Photo library images have a UUID identifier as _assetName (e.g 64EF5A48-2E96-4AB2-A79B-AAB7E9116E3D)
+        // SF symbol and bundle images have the actual symbol name as _assetName (e.g chevron.backward)
+        private func isPhotoLibraryImage(_ image: UIImage) -> Bool {
+            guard let assetName = image.imageAsset?.value(forKey: "_assetName") as? String else {
+                return false
+            }
+
+            if assetName.isEmpty { return false }
+            if image.isSymbolImage { return false }
+            if isAssetsImage(image) { return false }
```

Comment:
I think this method call should be an `or` operation with `isPhotoLibraryImage` since they are checking different things

Reply:
nvm, `isAssetsImage` is still checked next to `maskAllImages`, made the comment before reaching that part